internal/libs/mathslib/calc_test.go

```diff
@@ -4,9 +4,18 @@ import (
 	"math/big"
 	"testing"
 
+	"github.com/stretchr/testify/require"
+
+	"github.com/smartcontractkit/chainlink-ccip/pkg/types/ccipocr3"
+
 	"github.com/stretchr/testify/assert"
 )
 
+var (
+	SolChainSelector = ccipocr3.ChainSelector(16423721717087811551)
+	EvmChainSelector = ccipocr3.ChainSelector(16015286601757825753)
```

Comment:
Instead of these random looking numbers you can import the exact selectors from the chain-selectors lib